lib/cli.js

```diff
@@ -182,6 +183,19 @@ const cli = {
 
             log.info(JSON.stringify(fileConfig, null, "  "));
             return 0;
+        } else if (currentOptions.info) {
+            return new Promise(resolve => {
```

Comment:
While this is now testable, I don't think we want to have this be a conditionally async method. In this iteration, it'll return a promise when t's called with `currentOptions.info` and will be synchronous otherwise. In case you missed it, please check out [this comment](https://github.com/eslint/eslint/pull/12060#issuecomment-519339119) and let us know what you think!

Reply:
I just wonder why reinventing the wheel when everything is already cooked up and ready to use. Writing our own utility would be a pain as we need to ensure cross-platform compatibility while executing those shell commands. IMO leaving this as of now causes no harm :thinking:

Reply:
I wonder why reinventing the wheel when everything is already cooked up and ready to use. Writing our own utility would be a pain as we need to ensure cross-platform compatibility while executing those shell commands. IMO leaving this as of now causes no harm :thinking:

Reply:
What's tricky about the current proposed changes are that they add asynchronous code to something that has, up to this point, been synchronous. My concern is more about making sure we're not breaking anything, and I think having a method that, under some conditions returns a promise, is prone to bugs. 

As an alternative, we could explore making this method consistently return a promise, but we do need to make sure that it doesn't break anything for consumers. I didn't think that `lib/cli.js` was part of the public API, but [this comment](https://github.com/eslint/eslint/blob/master/lib/cli.js#L135-L138) definitely makes me want to double check with the rest of the team.